godot-macros/src/class/data_models/func.rs

```diff
@@ -637,6 +640,19 @@ fn make_call_context(class_name_str: &str, method_name_str: &str) -> TokenStream
     }
 }
 
+/// Returns a type annotation for the tuple corresponding to the signature declared on given ITrait method,
+/// allowing to validate params for a generated method call at compile time.
+///
+/// For example `::godot::private::virtuals::Node::Sig_physics_process` is `(f64, )`,
+/// thus `let params: ::godot::private::virtuals::Node::Sig_physics_process = ();`
+/// will not compile.
+fn make_tuple_sig_annotation(trait_base_class: &Ident, method_name: &Ident) -> TokenStream {
```

Comment:
```suggestion
fn make_sig_tuple_annotation(trait_base_class: &Ident, method_name: &Ident) -> TokenStream {
```
To be consistent with variable name.